tests/lib/rules/no-side-effect-cp.js

```diff
@@ -228,6 +242,55 @@ ruleTester.run('no-side-efffect-cp', rule, {
       errors: [{
         message: MESSAGE
       }]
+    },
+    // This test must come before any others that `import Ember from 'ember'`
+    // to illustrate the problem of the import binding being saved in module
+    // scope.
+    // see #91
+    {
+      code: `
+      import Ember from 'ember';
+      import SomethingElse from 'something-else';
+      const { set } = Ember;
```

Comment:
Can we tab these lines in to line them up with the code below. It's a bit hard to read like this.